### ISSUE TITLE
ifcwall class - error

### ISSUE THREAD

**Issue by adnanmakda** [IfcOpenShell/IfcOpenShell#836]
I tried assigning IFCwall class to an object and then exported it as IFC file. But when I tried opening it in Xbim as well as Revit, it shows an empty file. Most of the other classes are working well.
Thank you.

**Comment by Moult** [comment]
You might need to describe a little bit more :) Are you using code to export an IFC file? Are you using the BlenderBIM Add-on? Do you have a test file?

**Comment by adnanmakda** [comment]
I am using BlenderBim addon. So what I did was I used BlenderBim addon, did a quick setup and assigned Ifc class to the default cube. In the outliner, it showed that the change was made, after which I exported it. But when I opened the exported file in xbim, nothing showed up. So then I tried the same thing with pile class and that worked. I have attached the files below.
[test_1.zip](https://github.com/IfcOpenShell/IfcOpenShell/files/4466084/test_1.zip)
